data-raw/training-dec-2021.Rmd

```diff
@@ -368,17 +377,25 @@ g1 = ggplot(base_results) +
   geom_col(aes(dist_bands, Trips, fill = mode)) +
   scale_fill_manual(values = col_modes) + ylab("Trips")
 g1
+
+# Create the new scenario (first replace some car journeys with journeys by foot, then replace some of the remaining car journeys with journeys by bicycle)
```

Comment:
This comment is way too long. I suggest splitting it. By convention, lines of code including comments should be less than 80 characters.